server/graphql/invoice/src/lib.rs

```diff
@@ -7,7 +7,9 @@ pub mod invoice_queries;
 use self::invoice_queries::*;
 
 pub mod mutations;
-use self::mutations::{inbound_shipment, outbound_shipment, prescription, supplier_return};
+use self::mutations::{
+    customer_return, inbound_shipment, outbound_shipment, prescription, supplier_return,
```

Comment:
Is this an example of the back-end renaming you mentioned you would do?

Reply:
Yes has been renamed in a later PR 🙏